### PR TITLE
Added German translation

### PR DESCRIPTION
**Pull request by 13robin37**

### CODE REVIEW

File: custom_components/ui_lovelace_minimalist/translations/de.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        "step": {
+            "user": {
+                "title": "UI Lovelace Minimalist Einstellungen",
+                "description": "",
+                "data": {
+                    "language": "Sprache",
+                    "sidepanel_enabled": "Erstellung eines Eintrages in der Seitenleiste aktivieren.",
+                    "sidepanel_icon": "Icon Seitenleiste",
+                    "sidepanel_title": "Titel Seitenleiste",
+                    "theme": "Thema",
```
Comment (KGS501) [inline custom_components/ui_lovelace_minimalist/translations/de.json:13]: Would change it to "Stil" to make it more understandable

File: custom_components/ui_lovelace_minimalist/translations/de.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
```
Comment (KGS501) [inline custom_components/ui_lovelace_minimalist/translations/de.json:33]: Here aswell - "Stil"

**Review by KGS501 — APPROVED** [review]
Translation is understandable

File: custom_components/ui_lovelace_minimalist/translations/de.json (hunk 2 above)
Comment (13robin37) [reply custom_components/ui_lovelace_minimalist/translations/de.json:33]: The official translation right now is "Thema", check your profile settings in HA for example. Tried to keep it coherent.

Comment (KGS501) [reply custom_components/ui_lovelace_minimalist/translations/de.json:33]: Oh ok. In that case its fine the way it is👍

**Review by basbruss — APPROVED** [review]

**Pull request merged**